src/main/java/org/springframework/integration/kafka/core/BrokerAddress.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.springframework.integration.kafka.core;
+
+import org.springframework.util.StringUtils;
+
+/**
+ * Encapsulates the address of a Kafka broker
+ *
+ * @author Marius Bogoevici
+ */
+public class BrokerAddress {
+
+	public static final int DEFAULT_PORT = 9092;
+
+	private String host;
+
+	private int port;
```

Comment:
It can. I was trying to allow for setter-based initialization, because it's easier to understand, but eventually we'll have namespace support so that's moot.